modules/transports/core/vfs/src/main/java/org/apache/synapse/transport/vfs/PollTableEntry.java

```diff
@@ -499,6 +503,18 @@ public void setSubfolderTimestamp(String subfolderTimestamp) {
         this.subfolderTimestamp = subfolderTimestamp;
     }
 
+    public Long getMinimumAge() {
+        return minimumAge;
+    }
+
+    public Long getMaximumAge() {
+        return maximumAge;
+    }
+    
+    public boolean hasAgeCheck(){
```

Comment:
Are we utilizing this method?

Reply:
Method is effectively unused. Just removed it from this PR.